command/list.go

```diff
@@ -60,6 +60,10 @@ func (c *ListCommand) Run(args []string) int {
 			"No value found at %s", path))
 		return 1
 	}
+	if secret.WrapInfo != nil {
```

Comment:
Can we add `secret.WrapInfo.TTL != 0` also here?
Like: https://github.com/hashicorp/vault/blob/master/vault/request_handling.go#L57